credentials/apps/edx_credentials_extensions/edly_credentials_app/api/v1/views/tests/test_views.py

```diff
@@ -0,0 +1,124 @@
+"""
+Tests for edly_api Views.
+"""
+import json
+
+from django.urls import reverse
+
+from rest_framework.test import APITestCase
+
+from credentials.apps.catalog.tests.factories import ProgramFactory
+from credentials.apps.core.tests.factories import UserFactory
+from credentials.apps.core.tests.mixins import SiteMixin
+from credentials.apps.credentials.tests.factories import ProgramCertificateFactory
+
+JSON_CONTENT_TYPE = 'application/json'
+
+
+def get_test_image_as_base_64_encoded_string():
+    base64_header = 'data:image/png;base64,'
+    base64_data = (
+        '/9j/4AAQSkZJRgABAQEAYABgAAD/2wBDAKBueIx4ZKCMgoy0qqC+8P//8Nzc8P/////////////////////'
```

Comment:
This image content is still too large. In course discovery (course_discovery/apps/api/v1/tests/test_views/test_courses.py:924) we have something like this:

```python
# The API is expecting the image to be base64 encoded. We are simulating that here.
'image': 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY'
         '42YAAAAASUVORK5CYII=',
```

Reply:
Done